src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -1151,7 +1152,7 @@ public void BuildAppWithManagedResourceParser()
 		[NonParallelizable]
 		public void BuildAppWithManagedResourceParserAndLibraries ()
 		{
-			int maxBuildTimeMs = 10000;
+			int maxBuildTimeMs = TestEnvironment.IsRunningOnHostedAzureAgent ? 15000 : 10000;
```

Comment:
@dellis1972 @jonathanpeppers I've seen this take anywhere from 11-13 seconds so far on the Hosted Mac Internal pool, but we may want to reconsider the value change here based one some additional measurement?

Reply:
I'm sure it will be heavily dependent on the underlying machine/hard drive, so its not surprising its taking longer.

Reply:
Can we change this to not use a time?

Could it read the MSBuild log and fail if the managed parser wasn't used?

Reply:
It looks like we already have a number of asserts around a ManagedResourceParser related target running or not running at various points in this test. I'll keep this updated "magic number" in place for now and we can look to remove it or rework it at some point in the future.

https://github.com/xamarin/xamarin-android/blob/7ce9cdf6cb02e629ffe24de371cb224e5975dacd/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs#L1226